### ISSUE TITLE
About the NWPU-Crowd dataset

### ISSUE THREAD

**Issue by little-seasalt** [cxliu0/PET#19]
Dear author, I encountered some problems in the process of reproducing the NWPU data set: I screened the best models through the indicators of the validation set. After 1500 rounds of training, the indicators of the validation set reached about 73, and then I tested the test set on the best model, and submitted the results to the official website. I only got an MAE of 112. Do you know what the possible reasons are? Also, I would like to ask what is your approximate indicator on the validation set?Thank you for your answers in advance.

**Comment by cxliu0** [comment]
I think this issue is similar to the UCF-QNRF dataset. For large-scale datasets, it is important to ensure that the training patches contain a sufficient number of people. If many training patches are empty, the training supervision would be weak.

Regarding performance on the validation set, I remember the MAE is between 40 to 50.

**Comment by xzm429** [comment]
Does this mean that if I want to train a model that can recognize crowded crowds well, the training data images should try to keep the crowd dense enough to ensure that the patch contains a lot of features? I trained a model using the NWPU original dataset and found that for blurry and dense crowded crowds, point recognition is prone to produce a grid, which is obviously incorrect
![issue](https://github.com/cxliu0/PET/assets/105144975/505a86fd-b928-4ca1-878c-4effc22f716f)

**Comment by pk429** [comment]
> I think this issue is similar to the UCF-QNRF dataset. For large-scale datasets, it is important to ensure that the training patches contain a sufficient number of people. If many training patches are empty, the training supervision would be weak.
> 
> Regarding performance on the validation set, I remember the MAE is between 40 to 50.

Does this mean that if I want to train a model that can recognize crowded crowds well, the training data images should try to keep the crowd dense enough to ensure that the patch contains a lot of features? I trained a model using the NWPU original dataset and found that for blurry and dense crowded crowds, point recognition is prone to produce a grid, which is obviously incorrect
![issue](https://github.com/cxliu0/PET/assets/105144975/72a93a4f-4841-4d4b-a74c-ba52b93e8479)

**Comment by cxliu0** [comment]
Yes. If the model did not see crowded scenes during training, the model may not output good results in such scenarios during testing.
